lib/verbalize/action.rb

```diff
@@ -17,9 +17,20 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
+        @inputs = required_keywords
+        @optional_inputs = optional
+
         class_eval Build.call(required_keywords, Array(optional))
       end
 
+      def inputs
```

Comment:
My earlier comment got hidden by other changes:

I think we should name this `required_inputs` instead of `inputs`. It's a bit unintuitive that `input` takes both required and optional inputs, but `inputs` only returns one of the two.  We should use consistent naming and return all inputs here, or use appropriate method names if we are going to separate required/optional parameters.

Or, do all three - `inputs` returns required & optional parameters as a hash, while `(required|optional)_inputs` returns them individually.